example/Program.cs

```diff
@@ -6,151 +6,8 @@ namespace example
-        //terem
-        string RoomNumber;
-        //diák lista
-        HashSet<Student> Students = new HashSet<Student>();
-        public ClassRoom()
-        {
-        }
-
-        public ClassRoom(HomeRoomTeacher teach, string roomnbr, HashSet<Student> diakok)
-        {
-            this.Teacher = teach;
-            this.RoomNumber = roomnbr;
-            this.Students = studentClassSelector(diakok);
-        }
-
-        //paraméternek megkap egy student listát, és visszatér egy listával ami ebbe az osztályba jár
-        public HashSet<Student> studentClassSelector(HashSet<Student> studentsToSelectFrom)
-        {
-            HashSet<Student> selectedStudents = new HashSet<Student>();
-            foreach (Student student in studentsToSelectFrom) 
-            { 
-                if(this.Teacher.isTheirHomeRomeTeacher(student))
-                {
-                    selectedStudents.Add(student);
-                } 
-            }
-            return selectedStudents;
-        }
-        public override string ToString()
-        {
-            string roomproperties;
-
-            roomproperties = "Home Room " + RoomNumber + ", Home Room Teacher: " + Teacher.Name;
-            roomproperties  += "\nAktuális diák lista:\n";
-            foreach (Student s in Students)
-            {
-                roomproperties += s.Name + "\n";
-            }
-            return roomproperties.Trim();
-
-        }
-    }
-    class Student
-    {
-         public string Name { get; set; }
-         public DateTime? DOB { get; set; }
-         public string OMId { get; set; }
-         public string ClassLabel { get; set; } = "C";
-         public int ClassLevel { get; set; } = 12;
-
-        public Student() { }
-        /*
-         * Készítsen egy paraméteres konstruktort
-         */
-        public Student(string name, DateTime? dob, string omID, string label, int level)
-        {
-            this.Name = name;
-            this.DOB = dob;
-            this.OMId = omID;
-            this.ClassLabel = label;
-            this.ClassLevel = level;
-        }
-        //ird felül a totString-et, hogy 
-        public override string ToString()
-        {
-            return "A " + this.Name + " nevű tanuló a " + this.ClassLevel 
-                + this.ClassLabel + " osztályba jár."; 
-        }
-        //osztalytars-e - paraméter egy másik diák, visszatérési érték logikai
-        public bool isClassMate(Student otherStudent)
-        {
-            if (otherStudent == null)
-            {
-                return false;
-            }
-            return this.ClassLevel == otherStudent.ClassLevel && 
-                   this.ClassLabel== otherStudent.ClassLabel;
-        }
-
-        public override bool Equals(Object obj)
-        {
-            if (obj == null)
-                return false;
-
-            if (!(obj is Student))
-            {
-                return false;
-            }
-
-            // (Customer)obj;
```

Comment:
béna mert nem Customer hanem Student - watch your copy-paste - egyébként is ne hagyj szemét commeteket

Reply:
ne kötözködj, működik, nem?